doc/nrf/developing/code_base.rst

```diff
@@ -96,8 +96,15 @@ X, Y, and Z are the major, minor, and patch version, respectively.
 A special value of ``99`` for the patch version number is reserved for any revision in between releases.
 Tags without a suffix correspond to official releases tested and signed by Nordic Semiconductor engineers.
 A release candidate suffix ``-rcN`` is attached if the tag identifies a candidate instead of the actual release.
+
+.. dev_tag_definition_start
+
 In between releases, there might be development tags.
-These are identified by a ``-devN`` suffix.
+These are identified by a ``-devN`` suffix included after the |NCS| version, for example ``v1.9.99-dev1``.
```

Comment:
Should also 1.9.99-dev is done when done from main branch where as when done from release branch it would be 1.9.2-dev1 @carlescufi  agree ?

Reply:
I've added also an example for 1.9.2-dev1.